packages/components/src/templates/next/components/internal/Navbar/Navbar.tsx

```diff
@@ -26,211 +45,118 @@ export const Navbar = ({
+            {items.map((item, index) => (
+              <NavItem
+                key={`${item.name}-${index}`}
+                megaMenuRef={megaMenuRef}
+                ref={openNavItemIdx === index ? activeNavRef : null}
+                {...item}
+                onCloseMegamenu={handleClickOutside}
+                onClick={() => {
+                  setIsSearchOpen(false)
+                  setOpenNavItemIdx((currIdx) =>
+                    currIdx === index ? -1 : index,
+                  )
+                }}
+                isOpen={openNavItemIdx === index}
+                // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+                LinkComponent={LinkComponent}
+              />
+            ))}
           </ul>
-        </div>
 
-        {/* Spacer */}
-        <div className="flex-1" />
+          {/* Spacer */}
+          <div className="flex-1" />
+
+          {/* Search icon */}
+          {search && !isHamburgerOpen && (
+            <>
+              {isSearchOpen ? (
+                <button
+                  onClick={() => setIsSearchOpen(!isSearchOpen)}
+                  aria-label="Close search bar"
+                  className={icon({ className: "text-[1.5rem]" })}
+                >
+                  <BiX />
+                </button>
+              ) : (
+                <button
+                  onClick={() => setIsSearchOpen(!isSearchOpen)}
+                  aria-label="Open search bar"
+                  className={icon()}
+                >
+                  <BiSearch />
+                </button>
+              )}
+            </>
+          )}
 
-        {/* Search icon */}
-        {search && !isHamburgerOpen && (
-          <div className="my-auto block">
-            {isSearchOpen ? (
+          {/* Hamburger menu for small screens */}
+          <div className="flex lg:hidden">
+            {isHamburgerOpen ? (
               <button
-                onClick={() => setIsSearchOpen(!isSearchOpen)}
-                aria-label="Close search bar"
-                className="mt-[5px]"
+                onClick={() => {
+                  setIsHamburgerOpen(false)
+                  setOpenNavItemIdx(-1)
+                }}
+                className={icon({ className: "text-[1.5rem]" })}
+                aria-label="Close navigation menu"
               >
-                <BiX className="text-2xl" />
+                <BiX />
               </button>
             ) : (
               <button
-                onClick={() => setIsSearchOpen(!isSearchOpen)}
-                aria-label="Open search bar"
-                className="mt-[5px]"
+                onClick={() => {
+                  setIsHamburgerOpen(true)
+                  setIsSearchOpen(false)
+                }}
+                aria-label="Open navigation menu"
+                className={icon()}
               >
-                <BiSearch className="text-2xl" />
+                <HamburgerIcon />
```

Comment:
question: should we use `BiMenu` instead?

Reply:
Wanted to fit design and the sizing. Designers didn't use BiMenu so I never use lo HAHA